tgbot/tests/longpoll.rs

```diff
@@ -0,0 +1,83 @@
+use async_trait::async_trait;
+use dotenv::dotenv;
+use mockito::{mock, server_url, Matcher};
+use serde_json::json;
+use std::{
+    convert::Infallible,
+    sync::Arc,
+    time::{Duration, Instant},
+};
+use tgbot::{longpoll::LongPoll, types::Update, Api, Config, UpdateHandler};
+use tokio::{spawn, sync::Mutex, time::delay_for};
+
+struct Handler {
+    updates: Arc<Mutex<Vec<Update>>>,
```

Comment:
Get rid of `Mutex`

Reply:
Your suggestions?

Reply:
Oh, it is in test. It doesn't matter then